test/cluster/functionality.sh

```diff
@@ -28,6 +28,20 @@
 source "${OS_ROOT}/hack/lib/init.sh"
 os::util::environment::setup_time_vars
 
+# curl_es hits the Elasticsearch API from within a pod.
+function curl_es() {
+	local pod="$1"
+	local endpoint="$2"
+	shift; shift
+	local args=( "${@:-}" )
+
+	local secret_dir="/etc/elasticsearch/secret/"
+	oc exec "${pod}" -- curl --silent --insecure "${args[@]}" \
```

Comment:
You could, I just prefer to translate positional arguments into human-readable names ASAP.